modules/overview-of-kueue-resources.adoc

```diff
@@ -55,53 +66,28 @@ spec:
     effect: "NoSchedule"
 
 ----
-////
+
+Make sure that each flavor has the correct label selectors and taint tolerations so that workloads run on the expected nodes.
 
 ifndef::upstream[]
-ifdef::self-managed[]
-[NOTE]
-====
-In {productname-short} {vernum}, {org-name} supports only empty resource flavors.
-====
-endif::[]
-ifdef::cloud-service[]
-[NOTE]
-====
-In {productname-short}, {org-name} supports only empty resource flavors.
-====
+See the example resource-flavor configurations provided in link:{rhoaidocshome}{default-format-url}/managing_openshift_ai/managing_distributed_workloads#ref-example-kueue-resource-configurations_distributed-workloads[Example Kueue resource configurations].
```

Comment:
```suggestion
See the example `resource-flavor` configurations provided in link:{rhoaidocshome}{default-format-url}/managing_openshift_ai/managing_distributed_workloads#ref-example-kueue-resource-configurations_distributed-workloads[Example Kueue resource configurations].
```
or
```suggestion
See the example resource flavor configurations provided in link:{rhoaidocshome}{default-format-url}/managing_openshift_ai/managing_distributed_workloads#ref-example-kueue-resource-configurations_distributed-workloads[Example Kueue resource configurations].
```

Reply:
Removed the text entirely, it's not needed.